### ISSUE TITLE
About the inconsistency between the paper and the code.

### ISSUE THREAD

**Issue by l1n1999** [yangxh11/P2P-Net#1]
Hello, I see that you use Kullback-Leibler divergence function in the formula for feature regularization loss in your paper. Why you are using the cross entropy to calculate the feature regularization loss in the code?

**Comment by yangxh11** [comment]
Thanks. In the old version we did use KL divergence to regularize the feature learning, we then found it  more stable to use cross entropy during training. The KL divergence is equal to "A's entropy + CrossEntropy(A, B)",  we want to remove the first term.
